docs/imm-otel.md

```diff
@@ -0,0 +1,101 @@
+# Deploy the SignalFx Smart Agent and OpenTelemetry Collector for SignalFx Infrastructure Monitoring
+
+SignalFx Infrastructure Monitoring helps you gather metrics related to your system's performance. Metrics provide information about processes running inside the system, including counters, cumulative counters, and gauges.
+
+OpenTelemetry provides the libraries, agents, and other components that you need to capture telemetry from your services so that you can better observe, manage, and debug them. The following are just a few scenarios where you can use OpenTelemetry to monitor your solutions:
+
+* Creating custom async libraries 
+* Instrumenting metrics
+* Using multiple context propagation formats (B3, W3C) in parallel
+
+The OpenTelemetry Collector is a stand-alone service that can ingest metrics from various sources. The OpenTelemetry collector provides additional flexibility in configuration options. You can now configure integrations to send data to an OpenTelemetry Collector to centrally manage data sent to SignalFx.
```

Comment:
Not sure I would cover this here -- you are only talking about the collector below, so instrumentation is separate. I would likely skip all this but anything specific to Otel should be covered in Otel section below